src/components/studio-footer/StudioFooter.jsx

```diff
@@ -26,9 +28,9 @@ ensureConfig([
 ], 'Studio Footer component');
 
 const StudioFooter = ({
-  // injected
-  intl,
+  full,
```

Comment:
Same comment as https://github.com/openedx/frontend-app-course-authoring/pull/1258/files#diff-1c1e7c5912d6ff798a72899cde884c54211549e22fec1f469c8a5a4a854819e1R20 -- can this be named `fullWidth` or `fullSize` instead?

Reply:
Done: 80d05dc5967589925d08d432431abe413d7f9b57